resources/sass/app.scss

```diff
@@ -202,3 +202,23 @@
 .no-decoration {
     text-decoration: none !important;
 }
+
+.comment-row {
+    width: 100%;
+}
+
+.fa {
+    cursor: pointer;
+}
```

Comment:
c thấy em viết toàn bộ css vào 1 file này nhưng không phải ở màn hình nào cũng cần toàn bộ những css này, em phải tách file css ra, màn hình nào cần đoạn nào thì mới loà vào thôi